src/syntax_mapping.rs

```diff
@@ -155,6 +192,23 @@ fn basic() {
     );
 }
 
+#[test]
+fn git_xdg_config_home() {
+    use assert_cmd::Command;
+    let mut cmd = Command::cargo_bin("bat").unwrap();
+    let mut map = SyntaxMapping::builtin();
+
+    cmd.env("XDG_CONFIG_HOME", "/foo/bar");
+
+    map.insert("/foo/bar/git/config", MappingTarget::MapTo("Git Config"))
+        .ok();
+
+    assert_eq!(
+        map.get_syntax_for("/foo/bar/git/config"),
+        Some(MappingTarget::MapTo("Git Config"))
+    );
+}
```

Comment:
This does not test the new functionality at all.

1. `cmd.env(…)` will set up a new environment for a command which is yet to be run. But then you never actually run it. The whole `cmd`-related code here has no effect.
2. The test still passes without that code, because you are inserting `/foo/bar/git/config` into the mapping explicitly. It will obviously return the right value in that case.

Reply:
I removed the unit test. The other code looks good - thank you! I think it's okay to merge this without a test. Writing a proper test for this is nontrivial.